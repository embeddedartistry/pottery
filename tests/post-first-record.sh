pottery init
pottery post Formed a band, we formed a band, look at us, we formed a band
find docs/project-history -print
cat docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_post_FAKEUNIQUEID.md
