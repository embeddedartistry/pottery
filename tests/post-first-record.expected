pottery init
pottery post Formed a band, we formed a band, look at us, we formed a band
docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_post_FAKEUNIQUEID.md
find docs/project-history -print
docs/project-history
docs/project-history/1992
docs/project-history/1992/1992-01
docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_post_FAKEUNIQUEID.md
cat docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_post_FAKEUNIQUEID.md
Formed a band, we formed a band, look at us, we formed a band
